Tests/NIOCoreTests/ByteBufferTest.swift

```diff
@@ -3602,5 +3607,12 @@ extension ByteBufferTest {
             XCTAssertEqual(error as? Base64Error, .invalidCharacter)
         }
     }
-
+    
+    func testByteBufferDescription() {
```

Comment:
think it would be nice to add two other test cases:

- the empty ByteBuffer
- one where the hex bytes get truncated (i.e. more than 128 bytes or whatever the limit is)

Reply:
Thanks, added!